client/src/Pages/Register/style.css

```diff
@@ -0,0 +1,201 @@
+  display: flex;
+  justify-content: flex-start;
+}
+
+.ant-form .ant-form-item .ant-form-item-control .ant-form-item-control-input-content .ant-select {
+  width: 50%;
+}
+
+.ant-form .ant-form-item .ant-form-item-control .ant-form-item-control-input-content .ant-select-selector input {
+  height: 45px;
+  line-height: 45px;
+  text-align: left;
+  font-size: 18px;
+  font-weight: 300;
+}
+
+.ant-form .ant-form-item .ant-form-item-control .ant-form-item-control-input-content .ant-select-selector .ant-select-selection-placeholder {
+  text-align: left;
+  font-size: 18px;
+  font-weight: 300;
+}
+
+.ant-form .ant-form-item .ant-form-item-control input::placeholder {
+  font-size: 16px;
+  font-weight: 300;
+}
+
+.ant-form .ant-form-item .ant-form-item-control .ant-form-item-explain-error {
+  font-size: 16px;
+  font-weight: 300;
+  padding-bottom: 10px;
+}
+
+.ant-form .radio-label {
+  font-size: 16px;
+}
+
+.signup-btn {
+  color: white !important;
+  width: 200px;
+  height: 45px !important;
+  border: 1px solid #FDCB6E !important;
+  background-color: #FDCB6E !important;
+  border-radius: 3px !important;
+  line-height: 1 !important;
+  transition: all 0.25s;
+}
+
+.signup-btn:hover {
+  color: #FDCB6E !important;
+  background-color: transparent !important;
+  transition: all 0.25s;
+}
+
+@media only screen and (max-width: 1060px) {
+  .signup .left-section {
+    display: none;
+  }
+
+  .signup .signup-form-container .ant-form {
+    width: 100%;
+  }
+
+  .signup .signup-form-container .signin-link {
+    display: block;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+    color: inherit;
+    height: 45px;
+    line-height: 45px;
+    border: 1px solid #FDCB6E;
+    border-radius: 3px;
+    background-color: transparent;
+    padding-left: 0;
+    transition: all 0.25s;
+  }
+
+  .signup .signup-form-container .signin-link:hover {
+    color: white;
+    background-color: #FDCB6E;
+    transition: all 0.25s;
+  }
+
+  .signup-btn {
+    width: 100%;
+  }
+
+  .ant-row .ant-form-item {
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .signup-form-container .ant-form .ant-form-item .ant-form-item-label {
+    width: 100%;
+    text-align: left;
+  }
+
+  .signup-form-container .ant-form .ant-form-item .ant-form-item-control .ant-form-item-control-input-content .ant-select {
+    width: 100%;
+  }
+
+  .signup-form-container .ant-form .ant-form-item .ant-form-item-control {
+    width: 100%;
+    min-height: 45px;
+  }
+
+  .signup-form-container .ant-form-item .ant-form-item-control .ant-form-item-control-input .ant-select-selector {
+    height: 45px !important;
+  }
+  .signup-form-container .ant-form-item .ant-form-item-control .ant-form-item-control-input .ant-select-selector span {
+    line-height: 45px !important;
+  }
+}
+
+@media only screen and (max-width: 640px) {
+  .signup .signup-form-container .signin-link .ant-anchor .ant-anchor-ink {
+    display: none;;
+  }
```

Comment:
use [grid](https://ant.design/components/grid/#header) to make a responsive design